modules/nw-ovn-kubernetes-change-log-levels.adoc

```diff
@@ -86,16 +86,41 @@ configmap/env-overrides.yaml created
 [source,terminal]
 ----
 $ oc delete pod -n openshift-ovn-kubernetes \
---field-selector spec.nodeName=ip-10-0-217-114.ec2.internal -l app=ovnkube-node
+--field-selector spec.nodeName=ci-ln-3njdr9b-72292-5nwkp-master-0 -l app=ovnkube-node
 ----
 +
 [source,terminal]
 ----
 $ oc delete pod -n openshift-ovn-kubernetes \
---field-selector spec.nodeName=ip-10-0-209-180.ec2.internal -l app=ovnkube-node
+--field-selector spec.nodeName=ci-ln-3njdr9b-72292-5nwkp-master-2 -l app=ovnkube-node
 ----
 +
 [source,terminal]
 ----
-$ oc delete pod -n openshift-ovn-kubernetes -l app=ovnkube-master
+$ oc delete pod -n openshift-ovn-kubernetes -l app=ovnkube-node
+----
+
+. To verify all nodes for a specific pod, run the following command:
```

Comment:
I wasn't sure what you are verifying with this step - verifying the logging levels? Verifying logging for the nodes? It might just be me and my unfamiliarity with the subject though.

Reply:
These docs should address the confusion from https://issues.redhat.com/browse/OCPBUGS-18412

It is not obvious that the configmap environment variables are applied unless you check the logs themselves.